accumulo-store/src/main/java/gaffer/accumulostore/key/impl/RowIDAggregator.java

```diff
@@ -176,7 +176,7 @@ protected void findTop() throws IOException {
             topKey = new Key(workKey.getRowData().getBackingArray(), group.getBytes(CommonConstants.UTF_8),
                     elementConverter.buildColumnQualifier(group, topProperties),
                     elementConverter.buildColumnVisibility(group, topProperties),
-                    elementConverter.buildTimestamp(topProperties, workKey.getTimestamp()));
+                    elementConverter.buildTimestamp(topProperties));
```

Comment:
If you haven't specified a property for timestamp then you wont see the result anyway, if your using custom code its best as a general rule not to rely on anything not in the schema.